### PR TITLE
Feature qos metrics

### PR DESCRIPTION
**Pull request by thiessl**

### CODE REVIEW

File: src/main/java/ac/at/tuwien/infosys/visp/common/resources/OperatorQoSMetricsDTO.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
+package ac.at.tuwien.infosys.visp.common.resources;
+
+/**
+ * Created by thomas on 25.03.2017.
```
Comment (chochreiner) [inline src/main/java/ac/at/tuwien/infosys/visp/common/resources/OperatorQoSMetricsDTO.java]: please get rid of this autogenerated author notes

**Review by chochreiner — COMMENTED** [review]
looks fine apart from the author notes

**Comment by thiessl** [comment]
author notes removed.

**Comment by chochreiner** [comment]
added manually on master

**Pull request closed without merging**